servers/lumen/app/Http/Controllers/WaistCircumferenceController.php

```diff
@@ -6,45 +6,76 @@
 use Illuminate\Http\Request;
 use DB;
 use Predis\Autoloader;
+
 \Predis\Autoloader::register();
 
+
 class WaistCircumferenceController extends Controller
 {
     public function getAllTemporalWaistCircumferences()
     {
```

Comment:
Do you think the name waistCircumferenceQueryResultObj is better?

When I read waistCircumferenceQuery i think it is just the query